composer.json

```diff
@@ -24,7 +24,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.1",
+        "php": "^7.1 || ^8.0",
```

Comment:
Should we set PHP minimum version to `^7.4` for the new minor version?

Reply:
7.3 is still supported until end of next year. PHPUnit 9 supports 7.3 and 8 . Any specific reason to jump  to 7.4 right away?

Reply:
Alright, we can stick to PHP 7.3, because we don‘t want to use new features.